infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/plugins/LogReplicationPluginConfig.java

```diff
@@ -83,6 +86,8 @@ public LogReplicationPluginConfig(String filepath) {
 
             this.topologyManagerAdapterJARPath = prop.getProperty("topology_manager_adapter_JAR_path");
             this.topologyManagerAdapterName = prop.getProperty("topology_manager_adapter_class_name");
+
+            this.nodeIdFilePath = prop.getProperty("local_node_id_path");
```

Comment:
Can you add a comment on the method that specifies the format in which the node must be written in this file? for future reference..

Reply:
Added file format : 

> node_id=99F30C42-F96D-A0A7-5531-468E52926A01